### PR TITLE
Roller Bed Brake Fix

### PR DESCRIPTION
**Pull request by 1Sparky1**
Fixes being able to toggle roller bed brakes while you are the occupant.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Comment by 1Sparky1** [comment]
!bugfix

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -448,7 +448,7 @@
 		..()
 
 /obj/structure/bed/roller/AltClick(mob/user)
-	if(density && !use_check_and_message(user))
+	if(density && !use_check_and_message(user) && !(user == buckled))
```
Comment (FluffyGhoster) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:451]: You would probably want to use `user.incapacitated()`

**Comment by 1Sparky1** [comment]
After checking what use_check_and_message() does, this isn't needed

**Pull request closed without merging**